src/views/tutorial-view/components/variants-dropdown-disclosure/index.tsx

```diff
@@ -0,0 +1,47 @@
+import { useRouter } from 'next/router'
+import DropdownDisclosure, {
+	DropdownDisclosureLinkItem,
+} from 'components/dropdown-disclosure'
+import Text from 'components/text'
+import {
+	getVariantParam,
+	getVariantPath,
+	TutorialVariantOption,
+} from 'views/tutorial-view/utils/variants'
+import { VariantsDropdownDisclosureProps } from './types'
+import s from './variants-dropdown-disclosure.module.css'
+
+export function VariantsDropdownDisclosure({
+	variant,
+}: VariantsDropdownDisclosureProps) {
+	const { asPath } = useRouter()
+	// @TODO hook this into useVariants hook once data is wired
+	const activeOption = variant.options[0]
```

Comment:
🙌 ah sweet, i was wondering about the active state reflecting the URL, will look out for that in a later PR 👍